api/agentpb/agent.proto

```diff
@@ -72,7 +72,7 @@ message SetStateRequest {
     // Disables query examples for QAN Agents if true.
     bool disable_query_examples = 3;
     // Instructs QAN Agents to rotate query log file or table at this size if > 0.
-    int64 max_query_log_size = 4;
+    int64 max_slowlog_file_size = 4;
```

Comment:
That field will be used for other logs sources like MySQL slowlog table (a number of rows) or PostgreSQL query log (it doesn't call it "slowlog"). So a different name there is intensional.

Reply:
Note that comment above now also mentions "table"